data-raw/amazon_books.R

```diff
@@ -0,0 +1,8 @@
+## code to prepare `amazon-books` dataset goes here
+library(tidyverse)
```

Comment:
Please remove this line since we load the `tidyverse` at the beginning of the script already